examples/async_transcribe_local_media_file.js

```diff
@@ -15,14 +15,17 @@ const token = require('./config/config.json').access_token;
         metadata: "InternalOrderNumber=123456789",
         callback_url: "https://jsonplaceholder.typicode.com/posts",
         skip_diarization: false,
+        skip_punctuation: false,
+        speaker_channels_count: 1,
```

Comment:
1 is not the default value for speaker channels, we should not set this value in the example as it could change how stereo audios are transcribed.

Reply:
I see, I just wanted to display all the potential args. What do you suggest I set it to? Or should I comment it out?

Reply:
We discussed this and the default value should be `null`. There will need to be an update to the documentation in the near future to clarify this as well.